cmd/minikube/cmd/start.go

```diff
@@ -353,7 +355,27 @@ func runStart(cmd *cobra.Command, args []string) {
 		}
 	}
 
-	kubeconfig := node.Start(cc, n, existingAddons, true)
+	kubeconfig, err := node.Start(cc, n, existingAddons, true)
+	if err != nil && viper.GetBool(deleteOnFailure) {
+		out.T(out.Warning, "Cluster {{.name}} failed to start, deleting and trying again.", out.V{"name": cc.Name})
```

Comment:
Technically, it's a node start that failed. Should this output n.Name instead?

Reply:
makes sense, done